src/System.Device.Gpio/System.Device.Gpio.csproj

```diff
@@ -34,6 +34,7 @@
   <ItemGroup Condition="'$(TargetsWindows)' == 'true'">
     <!--Excluding Linux implementations-->
     <Compile Remove="**\*.Linux.cs" />
+    <Compile Remove="**\*.Unix.cs" />
```

Comment:
Why do we need to add the Unix suffix? I would either use the existing Linux one, or move all the Linux stuff to have the Linux suffix but not add a new one. The suffix is meant to represent the target OS platform, and today we still only target Linux and Windows, we don't have a separate target for Unix so I would remove this.

Reply:
This file is Unix because it uses Unix specific interop file coming from `Unix` folder. Just followed convention from Interop. You can use any Unix stuff on any Linux so don't see a problem

Reply:
the suffix is not intended to relate to the code used in the file, but instead to show which build configuration will the file be included in, and we just have a Linux and a win build, not a Unix one, so I would change it back to be Linux in order to be consistent and simple.